hack/e2e/kops.sh

```diff
@@ -54,6 +54,7 @@ function kops_create_cluster() {
     fi
     if test -f "$KOPS_PATCH_NODE_FILE"; then
       kops_patch_cluster_file "$CLUSTER_FILE" "$KOPS_PATCH_NODE_FILE" "InstanceGroup" "Node"
+      kops_patch_cluster_file "$CLUSTER_FILE" "$KOPS_PATCH_NODE_FILE" "InstanceGroup" "Master"
```

Comment:
Why would we need both Node & Master here?

Reply:
This part was discarded in this pr https://github.com/kubernetes-sigs/aws-efs-csi-driver/pull/1088/files